packages/connectivity/src/scp-cf/destination/destination-cache.ts

```diff
@@ -38,7 +38,7 @@ export interface DestinationCacheInterface {
 export class DefaultDestinationCache implements DestinationCacheInterface {
   cache: Cache<Destination>;
 
-  constructor(validityTime?: DateInputObject) {
+  constructor(validityTime?: number) {
```

Comment:
```suggestion
  constructor(validityTimeInMs?: number) {
```

Reply:
thank you for your review. I should change here also. I add `InMs` here too.